app/config.js

```diff
@@ -41,4 +41,4 @@ window.OPENSHIFT_VERSION = {
   kubernetes: "dev-mode"
 };
 
-window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:9000/dev-console";
+window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:" + (window.DEV_SERVER_PORT || '9001');
```

Comment:
But it's broken anyway since the project links will never work against the showcase app URL. We went from sometimes working to never working. You can customize it to point to another base URL if you need.